mobile-app/cypress/integration/functional/wallet/loans/loans.spec.ts

```diff
@@ -547,19 +552,18 @@ context('Wallet - Loans Payback Non-DUSD Loans', () => {
     cy.getByTestID('payment_token_card_dTU10').click()
   })
 
-  /* Paying dTU10 with sufficient dTU10 balance */
-  it('should borrow dTU10 to pay for previous dTU10 loan', () => {
-    getLoanTokenToPayback()
+  /* Paying dTU10 with dTU10 balance */
+  it('should topup dTU10 to pay for previous dTU10 loan', () => {
+    // getLoanTokenToPayback()
```

Comment:
we can remove this now?

Reply:
Updated! Removed (replaced with `Top up dTU10`)